src/helpers/flare.ts

```diff
@@ -88,3 +91,43 @@ export const getEndpointUrl = () => {
 
   return 'https://' + endpointUrl + FLARE_ENDPOINT_PATH
 }
+
+/**
+ * Searches current directory for project files
+ * @returns a set of file paths of project files
+ */
+export const getProjectFiles = async () => {
```

Comment:
Not sure if the AWS Lambda project files constant should be used for Cloud Run too, at least not the framework ones.

Reply:
I agree, cloud run now uses `FLARE_PROJECT_FILES`, and lambda uses `LAMBDA_PROJECT_FILES` which contains everything in `FLARE_PROJECT_FILES` and all the keys from `FRAMEWORK_FILES_MAPPING`.